Assets/MixedRealityToolkit/Sharing/Scripts/SDK/DiscoveryClientListener.cs

```diff
@@ -8,6 +8,8 @@
 // the SWIG interface file instead.
 //------------------------------------------------------------------------------
 
+using MixedRealityToolkit.Common.Extensions;
```

Comment:
Were these edits to SWIG generated files needed? I tend to be hesitant to manually edit these files. If so, this should be updated in the native repo as well, right?

Reply:
These are only temporary until the base DLL's are renamed.  hopefully the auto-gn should pick them up (else we'll review when importing the new DLL's / gen files.